app/decorators/pbs/person_decorator.rb

```diff
@@ -20,10 +20,18 @@ def roles_grouped
       end
     end
 
+    def siblings_in_context(context)
+      family_member_finder.family_members_in_context(context, kind: :sibling)
```

Comment:
ich habe die Logik in einen abgekapselten Service verschoben, weil sie doch an einigen verschiedenen Orten gebraucht wird und so einfacher getestet werden kann.